Assets/UI/Panels/citypaneloverview.lua

```diff
@@ -546,50 +546,75 @@ function ViewPanelAmenities( data:table )
+    
+    -- National Parks
     if GameCapabilities.HasCapability("CAPABILITY_CITY_HUD_AMENITIES_NATIONAL_PARKS") then
         CQUI_BuildAmenityBubbleInstance("ICON_UNITOPERATION_DESIGNATE_PARK", data.AmenitiesFromNationalParks, "LOC_PEDIA_CONCEPTS_PAGE_TOURISM_4_CHAPTER_CONTENT_TITLE");
     end
-    --War Weariness
+    
+    -- Starting Era
+	if(data.AmenitiesFromStartingEra > 0) then 
+        CQUI_BuildAmenityBubbleInstance("ICON_GREAT_PERSON_CLASS_SCIENTIST", data.AmenitiesFromStartingEra, "LOC_GAME_START_ERA");
+	end
+    
+    -- Improvements
+    CQUI_BuildAmenityBubbleInstance("ICON_IMPROVEMENT_PASTURE", data.AmenitiesFromImprovements, "LOC_IMPROVEMENT_NAME");
+    
+    -- War Weariness
     if GameCapabilities.HasCapability("CAPABILITY_CITY_HUD_AMENITIES_WAR_WEARINESS") then
-        CQUI_BuildAmenityBubbleInstance("ICON_UNITOPERATION_FORTIFY", (data.AmenitiesLostFromWarWeariness>0 and -data.AmenitiesLostFromWarWeariness or 0), "LOC_PEDIA_CONCEPTS_PAGE_COMBAT_3_CHAPTER_CONTENT_TITLE");
+        if data.AmenitiesLostFromWarWeariness > 0 then 
+            CQUI_BuildAmenityBubbleInstance("ICON_UNITOPERATION_FORTIFY", -data.AmenitiesLostFromWarWeariness, "LOC_PEDIA_CONCEPTS_PAGE_COMBAT_3_CHAPTER_CONTENT_TITLE");
+        end
     end
-    --Bankruptcy
+    
+    -- Bankruptcy
     if GameCapabilities.HasCapability("CAPABILITY_CITY_HUD_AMENITIES_BANKRUPTCY") then
-        CQUI_BuildAmenityBubbleInstance("ICON_NOTIFICATION_TREASURY_BANKRUPT", (data.AmenitiesLostFromBankruptcy>0 and -data.AmenitiesLostFromBankruptcy or 0), "LOC_PEDIA_CONCEPTS_PAGE_GOLD_4_CHAPTER_CONTENT_TITLE");
+        if data.AmenitiesLostFromBankruptcy > 0 then 
+            CQUI_BuildAmenityBubbleInstance("ICON_NOTIFICATION_TREASURY_BANKRUPT", -data.AmenitiesLostFromBankruptcy, "LOC_PEDIA_CONCEPTS_PAGE_GOLD_4_CHAPTER_CONTENT_TITLE");
+        end
     end
 
-    -- CQUI (AZURENCY) TODO : find the best icons for theses bubble
-    -- M4A Fix: data.AmenitiesFromDistricts was not always a number?
-    data.AmenitiesFromDistricts = tonumber(data.AmenitiesFromDistricts) or 0;
-    if (data.AmenitiesFromDistricts > 0) then
-        CQUI_BuildAmenityBubbleInstance("ICON_NOTIFICATION_TREASURY_BANKRUPT", Locale.ToNumber(data.AmenitiesFromDistricts), "LOC_HUD_CITY_AMENITIES_FROM_DISTRICTS");
-    end
+    -- Districts
+	data.AmenitiesFromDistricts = data.AmenitiesFromDistricts or 0;
+    CQUI_BuildAmenityBubbleInstance("ICON_DISTRICT_CITY_CENTER", data.AmenitiesFromDistricts, "LOC_DISTRICT_NAME");
 
+    -- Natural Wonders
     data.AmenitiesFromNaturalWonders = data.AmenitiesFromNaturalWonders or 0;
     if (data.AmenitiesFromNaturalWonders > 0) then
-        CQUI_BuildAmenityBubbleInstance("ICON_NOTIFICATION_TREASURY_BANKRUPT", Locale.ToNumber(data.AmenitiesFromNaturalWonders), "LOC_HUD_CITY_AMENITIES_FROM_NATURAL_WONDERS");
+        CQUI_BuildAmenityBubbleInstance("ICON_GREAT_PERSON_CLASS_MERCHANT", data.AmenitiesFromNaturalWonders, "LOC_NATURAL_WONDER_NAME");
```

Comment:
LOC_PEDIA_FEATURES_PAGEGROUP_NATURAL_WONDERS_NAME is the pluralized version of the word (_Natural Wonders_ instead of _Natural Wonder_)

Reply:
Same as above.